modules/users/client/components/AboutMe.component.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { NamespacesConsumer } from 'react-i18next';
+import '@/config/lib/i18n';
+import PropTypes from 'prop-types';
+import { limitTo, plainTextLength } from './utils/filters';
+
+export default function AboutMe({ profile, profileDescriptionToggle, user, appSettings }) {
+  return (<NamespacesConsumer ns={['user-profile']}>{ (t) => (<>
+    <section className="panel panel-default">
+      <header className="panel-heading">
+        {t('About me')}
+      </header>
+      <div className="panel-body">
+        {/* Short descriptions */}
+        {(profile.description.length < 2400 || profileDescriptionToggle) &&
+        <div dangerouslySetInnerHTML={{ __html: profile.description }} ></div>}{/* TODO Can we trust this HTML?*/}
+
+        {/* Long descriptions */}
+        {profile.description.length >= 2400 && !profileDescriptionToggle &&
+        <div
+          className="panel-more-wrap" ng-click="profileCtrl.profileDescriptionToggle=true">
+          <div className="panel-more-excerpt" setInnerHTML={limitTo(profile.description, 2400)}> </div>
```

Comment:
We'll also need to change this to `dangerouslySetInnerHTML`. Currently longer description isn't shown for me at all (Firefox) and I get a warning instead:

> Warning: React does not recognize the `setInnerHTML` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `setinnerhtml` instead. If you accidentally passed it from a parent component, remove it from the DOM element.`

Reply:
I am not sure how to deal with that, I am getting an error while changing it to dangerouslySetInnerHTML.
Can only set one of `children` or `props.dangerouslySetInnerHTML`.
I also don't know whether we should sanitize that HTML or we can trust it.

Reply:
There are a few sanitization libraries around. I've stumbled upon [DOMPurify](https://github.com/cure53/DOMPurify) and [sanitize-html](https://github.com/punkave/sanitize-html). Even if we trust the user data, it might still be a good security measure to sanitize it in the frontend.

@agatatalita What is your error?

Reply:
I was getting error: "Can only set one of `children` or `props.dangerouslySetInnerHTML`."
It was because I left a space in the div with property dangerouslySetInnerHTML.
`<div dangerouslySetInnerHTML={{__html: sanitizeHtml(profile.description)}}> </div>`

Now it works and I used sanitize-html.